src/components/Menu/Category.jsx

```diff
@@ -3,13 +3,11 @@ import React, { useState } from 'react';
 
 export default function Category(props) {
     const [gategoryName] = useState(props.gategoryName)
-//      const {
-//     chooseCategory,
-//     currentProducts
-//   } = props;
+    //  const {chooseCategory, currentProducts} = props;
+    //var element = document.getElementById("element");
     return (
-        <div className="element"
-            onClick={() => props.chooseCategory(gategoryName)}
+        <div className="element" id="element"
+            onClick={() => props.chooseCategory(gategoryName) && document.getElementById("element").classList.add("active")}
```

Comment:
lets try to avoid getElementById. I suppose we can use css as a props inside our component. Furthermore inside component should be state like isActive and in such case we should apply some specific css